stylesheets/partials/page-specific/pets-list.scss

```diff
@@ -0,0 +1,52 @@
+// TODO - MISSING COLOR in Variables
+$gray: #808080;
```

Comment:
Import the colors.scss and add anything you need there. $ccc and $other-gray need to be renamed to something meaningful.